features/step_definitions/folder_steps.rb

```diff
@@ -0,0 +1,37 @@
+Given(/^I already have the Git autocompletion symlink$/) do
+  FileUtils.rm File.expand_path('~/.config/fish/completions/git.fish')
+  FileUtils.symlink 'foo',
+                    File.expand_path('~/.config/fish/completions/git.fish')
+end
+
+
+Given(/^I have a different Git autocompletion file$/) do
+  FileUtils.rm File.expand_path('~/.config/fish/completions/git.fish')
+  IO.write File.expand_path('~/.config/fish/completions/git.fish'),
```

Comment:
You are repeating `File.expand_path('~/.config/fish/completions/git.fish')` a ton of times. Lets throw that in a constant in  `env.rb`